plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/partition/HiveHudiPartitionInfo.java

```diff
@@ -38,8 +37,6 @@
 public class HiveHudiPartitionInfo
         implements HudiPartitionInfo
 {
-    private static final Logger log = Logger.get(HiveHudiPartitionInfo.class);
```

Comment:
Make the cleanup in a separate commit.

Reply:
This was already removed by some other commit.